src/components/Table.module.css

```diff
@@ -0,0 +1,3 @@
+.tableBackground {
+  background-color: lightgoldenrodyellow;
```

Comment:
Please create a new folder styles where you can store all your different styles per components. This folder could be at the same level as pages and components are